core/domain/classifier_services.py

```diff
@@ -124,6 +124,35 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def check_classification_condition(state):
```

Comment:
@seanlip . Do you think the dead code is a big issue here? I've got a subsequent PR which is already hanging - The abstracting training one. This is will be used in that function. Does it make a difference whether its a property on State or the way it is?